modules/ws/src/test/resources/ws-consumer-wsdl-null-wsdl-location.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+	xmlns:ws="http://www.mulesoft.org/schema/mule/ws"
+	xsi:schemaLocation="
+        http://www.mulesoft.org/schema/mule/ws http://www.mulesoft.org/schema/mule/ws/current/mule-ws.xsd        
+        http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+        http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd">
+
+	<http:request-config name="HTTP_Request_Configuration"
+		protocol="HTTPS" host="fake.com" port="80">
+	</http:request-config>
+
+
+	<ws:consumer-config name="Web_Service_Consumer"
+		service="TestService" port="TestPort" serviceAddress="http://fake.com/services/Test"
+		connectorConfig="HTTP_Request_Configuration"
+		useConnectorToRetrieveWsdl="true" />
```

Comment:
This was removed and shouldn't be valid. If the test asserts the configuration failed to load, it should make sure it didn't fail because of errors like this one.